easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenJPEG'
+version = '2.3.0'
+
+homepage = 'http://www.openjpeg.org/'
+description = """OpenJPEG is an open-source JPEG 2000 codec written in
+ C language. It has been developed in order to promote the use of JPEG 2000,
+ a still-image compression standard from the Joint Photographic Experts Group
+ (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as
+ a JPEG 2000 Reference Software."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://github.com/uclouvain/openjpeg/archive'
```

Comment:
@murfreesboro Do you mind making this a one-liner? There's little point is spreading this across multiple lines if there's only a single entry...

```python
source_urls = ['https://github.com/uclouvain/openjpeg/archive']